tutorials/tour/_posts/2017-02-13-tour-of-scala.md

```diff
@@ -38,10 +38,7 @@ A [local type inference mechanism](local-type-inference.html) takes care that th
 
 ## Scala is extensible ##
 
-In practice, the development of domain-specific applications often requires domain-specific language extensions. Scala provides a unique combination of language mechanisms that make it easy to smoothly add new language constructs in the form of libraries:
-
-* any method may be used as an [infix or postfix operator](operators.html)
-* [closures are constructed automatically depending on the expected type](automatic-closures.html) (target typing).
+In practice, the development of domain-specific applications often requires domain-specific language extensions. Scala provides a unique combination of language mechanisms that make it easy to smoothly add new language constructs in the form of libraries. Any method may be used as an [infix or postfix operator](operators.html).
```

Comment:
Even in DSLs I thought people prefer to throw in the `.` these days. Although I don't see any mention in Dotty of doing anything about postfix ops, so maybe they are less dead than I thought... Anyway I personally prefer a dot before my op over a semicolon after it.